app/Models/User.php

```diff
@@ -26,6 +26,16 @@ class User extends Authenticatable
         'is_admin',
         'is_active'
     ];
+    
+    protected $guarded = [
+        'is_admin',
+        'is_active'
+    ];
+
+    public function Task()
```

Comment:
```suggestion
    public function tasks()
```
function lower case, hasMany => tasks (plural)